lib/fastlane/actions/typetalk.rb

```diff
@@ -0,0 +1,47 @@
+module Fastlane
+  module Actions
+    class TypetalkAction
+      def self.run(params)
+        options = {
+            message: nil,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.